index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link s rel="stylesheet" href="dist/css/main.css" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon" />
+    <title>Card</title>
+  </head>
+  <body>
+    <main>
+      <section class="center">
+        <!-- Circles-->
+        <div class="background">
+          <img
+            class="back1"
+            src="dist/images/bg-pattern-top.svg"
+            alt="picture-top"
+          />
+          <img
+            class="back2"
+            src="dist/images/bg-pattern-bottom.svg"
+            alt="picture-bottom"
+          />
+        </div>
+
+        <!-- Card -->
+        <article class="card">
+          <!-- Card-Head -->
+          <div class="card-head"></div>
+
+          <!-- Card-Info -->
+          <div class="card-info">
+            <div class="img">
+              <img src="dist/images/image-victor.jpg" alt="picture-profile" />
+            </div>
+            <h2 class="name">
+              Victor Crest
+              <span class="age">26</span>
+            </h2>
+            <div class="city">London</div>
+          </div>
+
+          <!-- Card-Footer -->
+          <div class="card-footer">
+            <div class="item">
+              <h4 class="value">80K</h4>
+              <h5 class="meta">Followers</h5>
+            </div>
+            <div class="item">
+              <h4 class="value">803K</h4>
+              <h5 class="meta">Likes</h5>
+            </div>
+            <div class="item">
+              <h4 class="value">1.4K</h4>
+              <h5 class="meta">Photos</h5>
+            </div>
+          </div>
+        </article>
+      </section>
+    </main>
+    <div id="test">
+      <img src="dist/images/test.jpg" alt="" />
```

Comment:
```suggestion
      <img src="dist/images/test.jpg" alt="Test image" />
```
Ne ostavljamo prazan alt!